RELEASE.md

```diff
@@ -116,7 +116,7 @@ Once the PR is merged, immediately update the tag to point to HEAD.
 
 ```shell
 git tag $BUILDER_TAG -f
-git push origin $BUILDER_TAG -f
+git push upstream $BUILDER_TAG -f
```

Comment:
Let's keep this to `origin`, even if we have `upstream` forks, since this is public documentation and should apply for forks too, I'd say

Reply:
alright. changed.